R/pkgs/report_generation/inst/rmarkdown/templates/state_report/skeleton/skeleton.Rmd

```diff
@@ -282,7 +282,8 @@ ARmap3 <- plot_geounit_attack_rate_map(cum_inf_dates,
                             shp,
                             scenariolabel = config$report$formatting$scenario_labels[1],
                             display_date = config$report$formatting$display_dates[3],
-                            viridis_palette = "cividis")
+                            viridis_palette = "cividis",
+                            popnodes = config$spatial_setup$popnodes)
```

Comment:
I thought we weren't putting config in these directly?

Reply:
we have been...